app/controllers/movies_controller.rb

```diff
@@ -1,5 +1,9 @@
 class MoviesController < ApplicationController
   def show
     @movie = Movie.find(params[:id])
+    @movieseances = Seance
+                    .where(movie_id: params[:id])
+                    .where(starts_at: DateTime.current..Date.today + 7.days)
```

Comment:
This might be simplified to:
```suggestion
                    .where(starts_at: DateTime.current..7.days.from_now)
```